exchanges/newexchange.md

```diff
@@ -0,0 +1,1061 @@
+# GoCryptoTrader package Exchanges
```

Comment:
Please git mv this to docs/ADD_NEW_EXCHANGE.md and add a link to it in the exchanges/README.md file